src/index.html

```diff
@@ -4,9 +4,47 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="../styles.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="
+      month
+      month--start-day-thurs
+      month--length-29"
+      >
```

Comment:
```suggestion
    <div 
      class="month month--start-day-thu month--length-29"
    >
```

Reply:
I thought if there are more than two classes it was best to use separate lines?